include/alpaka/mem/view/Traits.hpp

```diff
@@ -138,9 +180,24 @@ namespace alpaka
     //! \return The pitch in bytes. This is the distance in bytes between two consecutive elements in the given
     //! dimension.
     template<std::size_t Tidx, typename TView>
-    ALPAKA_FN_HOST auto getPitchBytes(TView const& view) -> Idx<TView>
+    [[deprecated]] ALPAKA_FN_HOST auto getPitchBytes(TView const& view) -> Idx<TView>
     {
+#if BOOST_COMP_CLANG || BOOST_COMP_GNUC
+#    pragma GCC diagnostic push
+#    pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#endif
         return trait::GetPitchBytes<DimInt<Tidx>, TView>::getPitchBytes(view);
+#if BOOST_COMP_CLANG || BOOST_COMP_GNUC
+#    pragma GCC diagnostic pop
+#endif
+    }
+
+    //! \return The pitches in bytes. This is the distance in bytes between two consecutive elements in the given
+    //! dimension.
+    template<typename TView>
+    ALPAKA_FN_HOST auto getPitchesInBytes(TView const& view) -> Vec<Dim<TView>, Idx<TView>>
+    {
+        return trait::GetPitchesInBytes<TView>{}(view);
```

Comment:
Let the bikeshedding begin. I am open to name suggestions for this function.

Reply:
`getByteDistance`?

Reply:
I would prefer if it had the word `pitch` or `stride` in it. `getByteDistance` sounds too generic to me.

Reply:
Right, this is similar to what @psychocoderHPC suggested. In principle, `getPitches` is OK.

I got quite confused some time ago, because it was not clear to me that the pitch was expressed in numbers of bytes and not in number elements, `getExtents` for example returnes the number of elements in each dimension. So to make the distinction clear, I preferred `getPitchBytes` in the past.

By now, I have a better understanding of what a pitch is, and in a CUDA, alpaka, picongpu, or image context, it has a clear meaning to me. Still, there is some overlap with the word stride and `std::mdspan` expresses the stride in number of elements. So, defensively, I opted for `getPitchesInBytes`, to make it very clear what is intended. The name is a bit long but clearer than `getBytePitches`, since we are not returning the pitches of a byte, but of a view in bytes :)

I am still ok with naming it `getPitches` if everybody wants so, since that is more consistent with `getExtents` and `getOffsets`. I would then make sure the documentation indicates clearly that the values are in bytes.

Reply:
Any other opinions? I have @j-stephan for `getPitches`, me for `getPitchesInBytes` and @psychocoderHPC for both. I need a tiebreaker.

Reply:
I like that `getPitchesInBytes` is more explicit and that `getPitches` is shorter...

If "pitch" is universally measured in bytes, I would favour `getPitches`.

The [wikipedia page](https://en.wikipedia.org/wiki/Stride_of_an_array) on array strides mixes up stride _vs_ pitch and elements _vs_ bytes, so it's not very helpful.

According to ChatGPT 3.5
> In a 2D array, the term "pitch" typically refers to the number of elements or bytes between the start of one row and the start of the next row in memory. It's often used in the context of optimizing memory access patterns, especially in languages like C or C++.

Again not helpful.

Given the overall confusion, I would go with `getPitchesInBytes`.

Reply:
Thx a lot for this analysis :D I would like to stick with `getPitchesInBytes` then.